### PR TITLE
SfSticky

### PR DESCRIPTION
**Pull request by przspa**
# Related issue
Create a sticky wrapper for components

# Scope of work
Created a sticky wrapper component without using position sticky.

# Screenshots of visual changes
sticky

# Checklist

- [x] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/atoms/SfSticky/SfSticky.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import '../../../css/variables';
+
+.sf-sticky {
+ position: sticky;
```
Comment (leomp12) [inline packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: Indentation isn't correct here.
I'm also not sure if it's a good idea to use `position: sticky`, it's not fully supported yet, don't you think is better to handle sticky position with JS?

Comment (leomp12) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: https://caniuse.com/#search=sticky

Comment (przspa) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: Ok, you have right, I be convinced that support is greater. My mistake, I fix it

Comment (przspa) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: @leomp12 I looked can i use again. Sticky doesnt work on IE and browesrs wersion before 2016. All partial support not appear to suport sticky table headers but we doesyn't .  It's to small support for us?

Comment (leomp12) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: @przspa imo it's important to be fully supported from IE11

Comment (leomp12) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: in general, modern libraries maintain support from IE11, and position sticky isn't supported...
you think we can discard IE11? wdyt @filrak ?

Comment (filrak) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: IE11 is still supported by Microsoft therefore we should support it at least at the level that could be transpiled via some postcss plugins. Not sure if it's available for sticky property

Comment (przspa) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: @leomp12 @filrak ok then I do that by JS :)

Comment (przspa) [reply packages/vue/src/components/atoms/SfSticky/SfSticky.scss]: @filrak @leomp12 now should be worked

**Comment by leomp12** [comment]
I think I should have suggested this before, I'm sorry...

**Comment by przspa** [comment]
@leomp12 I have idea. I checked computed style position and when browser doesn't support sticky then we have position static 👍  wdyt?
detect

**Comment by filrak** [comment]
As discussed - add ssr support with next PR

**Pull request merged**